Reneiski_Homework1/ReneiskiSelenium11/PageObjects/CheckBoxPage/CheckBoxPage.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium;
+
+namespace ReneiskiSelenium11.PageObjects.CheckBoxPage
+{
+    public class CheckBoxPage : BasePage
+    {
+        public CheckBoxPage(IWebDriver _driver) : base(_driver)
+        {
+        }
+
+        const string expectedText = "You have selected :\r\nhome\r\ndesktop\r\nnotes\r\ncommands\r\ndocuments\r\nworkspace\r\nreact\r\nangular\r\nveu\r\noffice\r\npublic\r\nprivate\r\nclassified\r\ngeneral\r\ndownloads\r\nwordFile\r\nexcelFile";
+
+        private IWebElement result => _driver.FindElement(By.Id("result"));
```

Comment:
privtate methods should go after public. please correct everywhere

Reply:
Fixed